auth0_flutter/test/web/auth0_flutter_web_test.dart

```diff
@@ -160,6 +161,32 @@ void main() {
     expect(params.max_age, null);
   });
 
+  test('loginWithRedirect supports appState parameter', () async {
```

Comment:
I ran the tests using `flutter test --platform=chrome --tags=browser -v` but I didn't see my test name in the verbose output? Is there a way to run this test locally?